web/src/app/rewardz/components/dashboard/rewards/rewards.component.ts

```diff
@@ -92,13 +117,14 @@ export class RewardsComponent extends NotificationComponent implements OnInit {
                     globalThis.wallet.script.SpoRewardClaim = rwd;
                     globalThis.wallet.script.SpoRewardClaim.recipient_stake_addr = globalThis.wallet.sending_stake_addr;
                     globalThis.wallet.script.SpoRewardClaim.recipient_payment_addr = globalThis.wallet.sending_wal_addrs[0];
-                    this.claimSubscription.unsubscribe();
-                    this.claimSubscription = null;
                     this.infoNotification("Submitting token claim");
-                    this.restService.fakeClaimTokens(this.submitURL)
+                    this.restService.buildTokenClaimTx("1", "sporwc")
```

Comment:
as mentioned in the other file, lets use a global "customerID" variable until we sorted out the environment variable issue.
Also "sporwc" should go to a variable I think but it does not have to be an environment variable as it is not changing.